tc/sgx/trusted_worker_manager/enclave_untrusted/enclave_bridge/signup.h

```diff
@@ -24,19 +24,32 @@ namespace tcf {
         namespace enclave_data {
             // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
             tcf_err_t CreateEnclaveData(
-                const std::string& inOriginatorPublicKeyHash,
+                StringArray& outPublicEnclaveData,
+                Base64EncodedString& outSealedEnclaveData,
+                Base64EncodedString& outEnclaveQuote);
+            
+            tcf_err_t CreateEnclaveDataKME(
+                const std::string& inExtData,
+                const std::string& inExtDataSignature,
                 StringArray& outPublicEnclaveData,
                 Base64EncodedString& outSealedEnclaveData,
                 Base64EncodedString& outEnclaveQuote);
 
-            // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
-            tcf_err_t UnsealEnclaveData(
-                StringArray& outPublicEnclaveData);
+            tcf_err_t CreateEnclaveDataWPE(
+                const std::string& inExtData,
+                const std::string& inExtDataSignature,
+                const std::string& inKmeAttestaion,
```

Comment:
Change inKmeAttestaion to inKmeAttestation here and everywhere else below.

Reply:
Done